server/models/procedures/migration-process.sql

```diff
@@ -144,3 +144,132 @@ BEGIN
 END $$
 
 DELIMITER ;
+
+/**
+author: @lomamech
+date: 2021-09-16
+description: Add column cost_center_id and principal_center_id in Voucher Item
+*/
+
+DROP PROCEDURE IF EXISTS PostVoucher;
+DELIMITER $$
```

Comment:
This shouldn't be in `migration-process.sql`.  Just replace the procedure found in the `procedures/voucher.sql`  file.

Reply:
Thank you I had hesitations